pkg/products/codeready/reconciler.go

```diff
@@ -201,6 +201,11 @@ func (r *Reconciler) reconcileExternalDatasources(ctx context.Context, serverCli
 		return integreatlyv1alpha1.PhaseAwaitingComponents, nil
 	}
 
+	_, err = resources.CreatePostgresConnectivityAlert(ctx, serverClient, postgres)
```

Comment:
@aidenkeating This is what's needed for the sso reconcilers once using external db.

Reply:
@david-martin 👍 To avoid stepping on anyone's toes, shall I hold off on any merging of SSO until this gets merged. Or is it ok for whichever PR gets merged last to include the changes?

Reply:
@aidenkeating I'd keep an eye on this PR (and #372) when you're finalising the SSO changes PR. Best to work it out between each other at merge time to avoid something being left out.